docs/config/keys.md

```diff
@@ -112,34 +228,39 @@ prefix.
 Similar in concept to the `phys:` mapping described above, the `raw:` mapping
 is independent of the OS keyboard layout.  Raw codes are hardware and windowing system dependent, so there is no portable way to list which key does what.
 
-To discover these values, you can set [debug_key_events =
-true](lua/config/debug_key_events.md) and press the keys of
-interest.
+To discover these values, you can set [debug_key_events = true](lua/config/debug_key_events.md) in your config, this will log key events in the [debug overlay](lua/keyassignment/ShowDebugOverlay.md).
+
 
 You can specify a raw key value of 123 by using `key="raw:123"` in your config
 rather than one of the other key values.
 
-### Leader Key
+## Leader Key
 
 *Since: 20201031-154415-9614e117*
 
-A *leader* key is a a modal modifier key.  If leader is specified in the
-configuration then pressing that key combination will enable a virtual `LEADER`
-modifier.
+The *leader* key is a modal modifier key that takes 3 fields:
+
+**Fields:**
+
+- key (required)
+- mods (optional)
+- timeout_milliseconds (optional)
+  - default: 1000 ms (1 second)
+
+
+If leader is specified in the configuration then pressing that key combination will enable a virtual `LEADER` modifier.
+
+While `LEADER` is active, only defined key assignments that include `LEADER` in the `mods` mask will be recognized.<br>
+Other key presses will be swallowed and **NOT** passed through to the terminal.
 
-While `LEADER` is active, only defined key assignments that include
-`LEADER` in the `mods` mask will be recognized.  Other keypresses
-will be swallowed and NOT passed through to the terminal.
+`LEADER` stays active until a keypress is registered (whether it matches a key binding or not),
+or until it has been active for the duration specified by `timeout_milliseconds`, at which point it will automatically cancel itself.
 
-`LEADER` stays active until a keypress is registered (whether it
-matches a key binding or not), or until it has been active for
-the duration specified by `timeout_milliseconds`, at which point
-it will automatically cancel itself.
+Here's an example configuration using `LEADER`.
 
-Here's an example configuration using `LEADER`.  In this configuration,
-pressing `CTRL-A` activates the leader key for up to 1 second (1000
-milliseconds).  While `LEADER` is active, the `|` key (with no other modifiers)
-will trigger the current pane to be split.
+In this configuration,
+pressing `CTRL-A` activates the leader key for up to 1 second (1000 milliseconds).<br>
+While `LEADER` is active, the `'|'` key (with no other modifiers) will trigger the current pane to be split.
```

Comment:
This text says that no modifier is needed, but the configuration says `mods = 'LEADER|SHIFT'` why is that? Is it because many keyboard layouts need shift to produce `|` ?